Cargo.toml

```diff
@@ -20,8 +20,7 @@ rand_core = "0.6"
 rand = "0.8"
 rand_chacha = "0.3.1"
 tikv-jemallocator = "0.5.4"
-# wsts = { version = "8.1", default-features = false }
-wsts = { git = "https://github.com/stacks-network/wsts.git", branch = "feat/public-sign-ids" }
+wsts = { git = "https://github.com/stacks-network/wsts.git", default-features = false, rev = "cf395701cb1332e4d66e70d8baf60b1658c168b5" }
```

Comment:
This will be updated once https://github.com/stacks-network/wsts/pull/7 is merged and a new `wsts` version is released.

Reply:
Commit 1b9589df9c1ee299423bffdf8f74298d9f4464f2 is the latest on the PR branch if you want to test the Party API changes

Reply:
Thanks, I've updated this now. It seemed to work well running the integration test locally.

Reply:
Set to 9.0.0 :heavy_check_mark: